docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+  <references type="soft" candidates="718240" cleared="2858" enqueued="2832" dynamicThreshold="18" maxThreshold="32" />
+  <references type="weak" candidates="2321" cleared="142" enqueued="0" />
+  <references type="phantom" candidates="8" cleared="0" enqueued="0" />
+  <stringconstants candidates="9522" cleared="0"  />
+  <object-monitors candidates="7142" cleared="7066"  />
+</gc-op>
+<gc-op id="1222" type="classunload" timems="0.704" contextid="1154" timestamp="2021-02-26T11:17:37.030">
+  <classunload-info classloadercandidates="185" classloadersunloaded="13" classesunloaded="13" anonymousclassesunloaded="0" quiescems="0.000" setupms="0.644" scanms="0.043" postms="0.016" />
+</gc-op>
+<gc-end id="1223" type="global mark phase" contextid="1154" durationms="169.521" usertimems="1244.810" systemtimems="3.000" stalltimems="27.792" timestamp="2021-02-26T11:17:37.034" activeThreads="8">
+  <mem-info id="1224" free="1438646272" total="4294967296" percent="33">
+    <pending-finalizers system="30" default="0" reference="2832" classloader="0" />
+    <remembered-set count="2241440" freebytes="162738560" totalbytes="171704320" percent="94" regionsoverflowed="3" regionsstable="127" regionsrebuilding="0"/>
+  </mem-info>
+</gc-end>
+```
+
+Comparing the status of the memory at the start and end of this final `global mark phase` increment shows that:
+
+- As expected, the final global mark phase increment does not reclaim any free memory.
+- The remembered set metastructure has been rebuilt. The count has reduced and the amount of available memory in the metastructure has increased from 91% to 94%.
+- The number of overflow regions remains unchanged. The final global mark phase increment did not manage to rebuild any overflow regions.
+
+Following the final global mark increment, the global mark cycles completes and the GC ends the STW pause.
+
+```xml
+<cycle-end id="1225" type="global mark phase" contextid="1154" timestamp="2021-02-26T11:17:37.034" />
+<exclusive-end id="1226" timestamp="2021-02-26T11:17:37.034" durationms="170.186" />
+
+```
+
+The operations to create a record of object liveness across the heap, which began with the global *mark* cycle, is completed with a sweep phase. The sweep phase is triggered by the end of the global *mark* cycle to be included in the next partial GC cycle that runs.
+
+#### Sweep phase
+
+While the global *sweep* operation is logically associated with the global *mark* phase, it does not run in the same global *mark* cycle. Instead, the *sweep* operation runs in the same STW increment as the first partial GC cycle that runs after the completion of the global *mark* cycle. This can be seen in the following log excerpt.  After the log records the end of the global mark cycle it records a STW pause followed by a `partial gc` cycle of `id=1229`. The global *sweep* operation that runs after the the global *mark* phase is recorded in the `<gc-op>` element tagged as `id=1229`.
```

Comment:
You may add the purpose of sweep, which is dual:
- it may directly reclaim some memory by creating completely empty regions
- for those regions that still have live object, it builds information about occupancy and fragmentation that can be used later when PGC picks older regions for defragmenation.

Reply:
Updated to:

#### Sweep phase

The sweep operation has two objectives:

- to directly reclaim some memory by creating completely empty regions.
- to build information about occupancy and fragmentation for regions that still contain live objects. The next partial GC cycle uses this information to defragment older regions.